job-dsls/jobs/osbs/operators_osbs_seed_cloud_jobs.groovy

```diff
@@ -0,0 +1,202 @@
+
+
+prodComponent.each { Component ->
+
+    pipelineJob("${folderPath}/${Component}") {
+
+        parameters {
+            stringParam('BUILD_DATE', "${buildDate}")
+            stringParam('PROD_VERSION', "${prodVersion}")
+            stringParam('PROD_COMPONENT', "${Component}")
+            stringParam('OSBS_BUILD_TARGET', "${osbsBuildTarget}")
+            stringParam('CEKIT_BUILD_OPTIONS', "${cekitBuildOptions}")
+            stringParam('KERBEROS_PRINCIPAL', "${kerberosPrincipal}")
+            stringParam('OSBS_BUILD_USER', "${osbsBuildUser}")
+            stringParam('KERBEROS_KEYTAB', "${kerberosKeytab}")
+            stringParam('KERBEROS_CRED', "${kerberosCred}")
+            stringParam('IMAGE_REPO', "${imageRepo}")
+            stringParam('IMAGE_BRANCH', "${imageBranch}")
+            stringParam('IMAGE_SUBDIR', "${imageSubdir}")
+            stringParam('GIT_USER', "${gitUser}")
+            stringParam('GIT_EMAIL', "${gitEmail}")
+            stringParam('CEKIT_CACHE_LOCAL', "${cekitCacheLocal}")
+            stringParam('VERBOSE', "${verbose}")
+            stringParam('GITHUB_ORG_UNIT', "${githubOrgUnit}")
+            stringParam('BAMOE_BA_OPERTOR_BRANCH', "${bamoeBAOperatorBranch}")
+            stringParam('BAMOE_KOGITO_OPERTOR_BRANCH', "${bamoeKogitoOperatorBranch}")
+        }
+
+        definition {
+            cps {
+
+                script('''
+
+                      library 'jenkins-pipeline-shared-libraries'
+                      
+                      private void validateParameters(required, optionals){
+                          // Check if all required params are supplied
+                          for ( param in required ) {
+                              if(!param.value[0]){
+                                  error "${param.key} parameter is required but was not specified."
+                              }
+                          }
+                      }
+                      
+                      // Function to retrieve from the PROD_COMPONENT name the related repo name into the kiegroup org
+                      private String getOperatorRepoName(prodComponent){
+                          switch(prodComponent){
+                            case { it.endsWith('ba-operator') }:
+                                return 'kie-cloud-operator'
+                            case { it.endsWith('kogito-operator') }:
+                                return 'kogito-operator'
+                            default:
+                               error "${prodComponent} not supported."
+                          }
+                      }
+                      
+                      // Function to retrieve from the PROD_COMPONENT name the related repo branch
+                      private String getOperatorBranch(prodComponent){
+                          switch(prodComponent){
+                            case { it.startsWith('rhpam') }:
+                                return 'main'
```

Comment:
for kogito I guess it is not the main branch, it should be 1.13.x and for bamoe 1.13.x-blue

for ba-operator main stands for the next development cycle, so main branch is correct
and for. bamoe ba, it is release-v7.13.x-blue

Reply:
I added the constant RHPAM_KOGITO_OPERTOR_BRANCH for the Kogito Operator branch.

You can see it here https://github.com/kiegroup/kie-ci/pull/1407/files#diff-9d55ce9075a526a3684617ae54e1f15ee81673ba5c89794b916086a8ffacc4d9R69